tests/acl/test_acl.py

```diff
@@ -574,14 +588,111 @@ def stage(request, duthosts, rand_one_dut_hostname, tbinfo, is_macsec_enabled_fo
+
+    def check_active_active_port_status(duthost, ports, status):
+        logging.debug(f"Check mux status for ports {ports} is {status}")
+        show_mux_status_ret = show_muxcable_status(duthost)
+        logging.debug("show_mux_status_ret: {}".format(
+            json.dumps(show_mux_status_ret, indent=4)))
+        for port in ports:
+            if port not in show_mux_status_ret:
+                return False
+            elif show_mux_status_ret[port]['status'] != status:
+                return False
+        return True
+
+    def _config_the_active_active_dualtor(active_tor, standby_tor, ports, unconditionally=False):
+        active_side_commands = []
+        standby_side_commands = []
+        logging.info(f"Configuring {active_tor.hostname} as active")
+        logging.info(f"Configuring {standby_tor.hostname} as standby")
+        for port in ports:
+            if port not in active_active_ports:
+                raise ValueError(
+                    f"Port {port} is not in the active-active ports")
+            active_side_commands.append(f"config mux mode active {port}")
+            standby_side_commands.append(f"config mux mode standby {port}")
+
+        if not check_active_active_port_status(active_tor, ports, 'active') or unconditionally:
+            active_tor.shell_cmds(cmds=active_side_commands)
+        standby_tor.shell_cmds(cmds=standby_side_commands)
+
+        pytest_assert(wait_until(30, 5, 0, check_active_active_port_status, active_tor, ports, 'active'),
+                      f"Could not config ports {ports} to active on {active_tor.hostname}")
+        pytest_assert(wait_until(30, 5, 0, check_active_active_port_status, standby_tor, ports, 'standby'),
+                      f"Could not config ports {ports} to standby on {standby_tor.hostname}")
+
+        ports_to_restore.extend(ports)
+
+    ports_to_restore = []
+    _config_the_active_active_dualtor(
+        rand_selected_dut, rand_unselected_dut, active_active_ports)
+
+    yield
+
+    if ports_to_restore:
+        restore_cmds = []
+        for port in ports_to_restore:
+            restore_cmds.append(f"config mux mode auto {port}")
+
+        for duthost in duthosts:
+            duthost.shell_cmds(cmds=restore_cmds)
+
+
+def create_custom_acl_table_type(duthost):
+    duthost.copy(src=MULTI_BINDING_ACL_TABLE_TYPE_SRC_FILE, dest=MULTI_BINDING_ACL_TABLE_TYPE_DST_FILE)
+    duthost.command(f"sonic-cfggen -j {MULTI_BINDING_ACL_TABLE_TYPE_DST_FILE} -w")
+
+
+def remove_custom_acl_table_type(duthost):
+    duthost.shell("sonic-db-cli CONFIG_DB del \'ACL_TABLE_TYPE|MULTI_BINDING_ACL\'")
+
+
+@pytest.fixture(scope="class", autouse=True)
+def multi_binding_acl_table_type(duthosts, rand_selected_dut):
+    if is_multi_binding_acl():
+        create_custom_acl_table_type(rand_selected_dut)
+        yield
+        remove_custom_acl_table_type(rand_selected_dut)
+    else:
+        yield
+
+
+def is_sai_profile_multi_binding_enabled(duthost):
+    """
```

Comment:
I prefer enabling `SAI_ACL_MULTI_BINDING_ENABLED` if it's not enabled on Mellanox dualtor-aa

Reply:
It should enabled by default, else there should be some issue. So I added a logic to check whether it's enabled. If not, the case would be failed, and maybe some design issue should be created.